razor.sublime-syntax

```diff
@@ -0,0 +1,136 @@
+      embed_scope: source.cs.embedded.html
+      escape: $
+    - match: (@)(section)\b(?:\s+(\w+))?
+      captures:
+        1: punctuation.section.embedded.line.cshtml
+        2: source.cs.embedded.html keyword.other.cshtml
+        3: entity.name.section.cshtml
+      push:
+        - match: \{
+          scope: punctuation.section.block.begin.cshtml
+          set:
+            - match: \}
+              scope: punctuation.section.block.end.cshtml
+              pop: true
+            - include: razor_html
+    - match: (@)(functions)\b
+      captures:
+        1: punctuation.section.embedded.line.cshtml
+        2: source.cs.embedded.html keyword.other.cshtml
+      push:
+        - match: \{
+          scope: punctuation.section.block.begin.cshtml
+          set:
+            - meta_scope: source.cs.embedded.functions.cshtml
+            - match: \}
+              scope: punctuation.section.block.end.cshtml
+              pop: true
+            - include: scope:source.cs#code_block_in
+        - match: \s*(?!\{)(?=\S)
+          scope: invalid.illegal.expected-block.cshtml
+          pop: true
+    - match: (@)(using)\b(?!\s*\()
+      captures:
+        1: punctuation.section.embedded.line.cshtml
+        2: source.cs.embedded.html keyword.other.cshtml
+      embed: scope:source.cs
+      embed_scope: source.cs.embedded.html
+      escape: $
+    - match: '@@'
+      scope: constant.character.escape.cshtml
+    - match: (@)(?=(?:if|switch|for|foreach|using|try|lock)\b)
+      captures:
+        1: punctuation.section.embedded.cshtml
+      push:
+        - meta_content_scope: source.cs.embedded.html
+        - include: scope:source.cs#line_of_code
+        - match: ''
+          pop: true
+      with_prototype:
+        - include: razor_block_prototype
+    - match: '[\w-+]+@\w+' # ignore email addresses
+    - match: '@(?=(?:\w+\.)*\w+(?:[<\s]|$))'
+      scope: punctuation.section.embedded.line.cshtml
+      embed: scope:source.cs#line_of_code_in
+      embed_scope: source.cs.embedded.html
+      escape: $|(?=[<\s])
+    - match: '@(?=(?:\w+\.)*\w+\()'
+      scope: punctuation.section.embedded.line.cshtml
+      push:
+        - meta_content_scope: source.cs.embedded.html
+        - match: (?=[<\s])
+          pop: true
+        - include: scope:source.cs#line_of_code_in
+    - match: '@\('
+      scope: punctuation.section.embedded.begin.cshtml
+      push:
+        - meta_content_scope: source.cs.embedded.html
+        - match: \)
+          scope: punctuation.section.embedded.end.cshtml
+          pop: true
+        - include: scope:source.cs#line_of_code_in
+    #- match: '@(?=\w+(?:\.\w+)*)(?!\s*[+=|(])' # TODO: support a.b[2] etc.
+    #  comment: property/variable access
+    #  scope: punctuation.section.embedded.line.cshtml
+    #  #push: razor_block # TODO: stop at closing " etc.
+    - match: '@'
+      scope: punctuation.section.embedded.line.cshtml
+      push: razor_block
+
+  razor_block_prototype:
```

Comment:
Do you mind using "kabob-case" for context names? Also lines 120, 124, 134.

Reply:
done :)